### ISSUE TITLE
Maby this plugin can be replace with sublime-LSP

### ISSUE THREAD

**Issue by jingyuexing** [Pure-D/sublime-d#3]
this plugin can be developed with [LSP](https://github.com/sublimelsp/LSP),Like [LSP-Dart](https://github.com/sublimelsp/LSP-Dart)

**Comment by WebFreak001** [comment]
yeah serve-d can be used with it too, don't have a config for it ready though. If anyone makes it work, feel free to PR your config to the serve-d README
